demos/ruby-hello/ruby-hello

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Ah, snaps do not execute all by themselves. They have no bash, no dash, no rootfs. Snaps "execution environment" if you will is the core snap, that's why it exists. It holds all those common things, including `/usr/bin/env`.